src/core/lib/channel/channel_trace.cc

```diff
@@ -48,31 +48,37 @@ ChannelTrace::TraceEvent::TraceEvent(Severity severity, grpc_slice data,
       timestamp_(grpc_millis_to_timespec(grpc_core::ExecCtx::Get()->Now(),
                                          GPR_CLOCK_REALTIME)),
       next_(nullptr),
-      referenced_entity_(std::move(referenced_entity)) {}
+      referenced_entity_(std::move(referenced_entity)) {
+  memory_usage_ = sizeof(TraceEvent) + grpc_slice_memory_usage(data);
```

Comment:
This can be set in the initializer list.

Reply:
Done